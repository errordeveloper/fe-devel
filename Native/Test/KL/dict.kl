operator entry() {
  Scalar a[String];
  report "a = " + a;
  report "a.size = " + a.size;
}
