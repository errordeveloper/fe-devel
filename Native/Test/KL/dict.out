a = {}
a.size = 0
